ntoskrnl/mm/amd64/procsup.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * COPYRIGHT:       GPL, See COPYING in the top level directory
+ * PROJECT:         ReactOS kernel
+ * FILE:            ntoskrnl/mm/amd64/procsup.c
```

Comment:
As you're creating a new file, please do it with the "new" header :)

Reply:
This is actually code from @tkreuzer I can't relicence it just because I put it in a new file